packages/svelte2tsx/test/svelte2tsx/samples/$store-as-directive/expectedv2.ts

```diff
@@ -9,8 +9,8 @@
 ;
 async () => {
 
- { svelteHTML.createElement("div", {      });__sveltets_2_ensureTransition($transitionStore(svelteHTML.mapElementTag('div'),({ y: 100 })));__sveltets_2_ensureAction($actionStore(svelteHTML.mapElementTag('div')));__sveltets_2_ensureTransition($inStore(svelteHTML.mapElementTag('div')));__sveltets_2_ensureTransition($outStore(svelteHTML.mapElementTag('div')));__sveltets_2_ensureAnimation($animateStore(svelteHTML.mapElementTag('div'),__sveltets_2_AnimationMove));
- }};
+ {const $$action_0 = __sveltets_2_ensureAction($actionStore(svelteHTML.mapElementTag('div')));{ svelteHTML.createElement("div", __sveltets_2_union($$action_0), {      });__sveltets_2_ensureTransition($transitionStore(svelteHTML.mapElementTag('div'),({ y: 100 })));__sveltets_2_ensureTransition($inStore(svelteHTML.mapElementTag('div')));__sveltets_2_ensureTransition($outStore(svelteHTML.mapElementTag('div')));__sveltets_2_ensureAnimation($animateStore(svelteHTML.mapElementTag('div'),__sveltets_2_AnimationMove));
```

Comment:
I can only see tests for `$$action_0`. I think a test with multiple actions applied to a dom element would be good.

What happens if two actions expect the same `attribute` with a different type?

Reply:
A test with multiple actions is a good idea, yes. If actions conflict then, well, nothing we can do about that 😄 If they types conflict that likely means these actions aren't compatible at runtime, too.